src/Table/AbstractTable.php

```diff
@@ -263,10 +263,16 @@ public function delete(RowInterface $row)
      * @return RowInterface
      *
      */
-    public function newRow(array $cols = [])
+    public function newRow(array $colsVals = [])
```

Comment:
I thought you might be @yespire :-)

The idea with $cols vs $colsVals is that $cols is for populating row data, and $colsVals is for queries. If you can think of a better convention, let me know.